libs/profile/src/lib/components/profile/profile.component.html

```diff
@@ -56,14 +63,23 @@ <h2 class="mb-4 text-lg font-bold text-center">{{'account_name_new' | transloco}
              let-data="data"
              let-close="close">
   <div class="m-4">
-    <h2 class="mb-4 text-lg font-bold text-center">{{'password_new' | transloco}}</h2>
-    <input #password
-           id="password"
-           name="password"
-           type="password"
+    <h2 class="mb-4 text-lg font-bold text-center">{{'password_old' | transloco}}</h2>
+    <input #oldpassword
+           id="oldassword"
+           name="oldpassword"
+           type="oldpassword"
            class="w-full px-5 py-3 text-base placeholder-gray-300 transition duration-500 ease-in-out transform border border-blue-200 rounded-sm focus:outline-none focus:ring-2 focus:ring-blue-200 focus:ring-offset-2 focus:ring-offset-blue-200">
-    <button class="w-full p-4 my-4 text-center border border-blue-500 rounded-lg focus:outline-none focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-blue-300"
-            (click)="close({newPassword: password.value})">
+           <h2 class="mb-4 text-lg font-bold text-center">{{'password_new' | transloco}}</h2>
+          
+           <input #newpassword
+           id="newpassword"
+           name="newpassword"
+           type="newpassword"
+           class="w-full px-5 py-3 text-base placeholder-gray-300 transition duration-500 ease-in-out transform border border-blue-200 rounded-sm focus:outline-none focus:ring-2 focus:ring-blue-200 focus:ring-offset-2 focus:ring-offset-blue-200">
+           <div class="flex items-center w-full h-4 mb-4">
+            Das Password muss diese Anforderungen erfüllen: </div>
+           <button class="w-full p-4 my-4 text-center border border-blue-500 rounded-lg focus:outline-none focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-blue-300"
+            (click)="close({oldPassword: oldpassword.value},{newPassword: newpassword.value})">
```

Comment:
Hier gibst du zwar das alte Passwort mit in die "onChangePassword()" Methode in der profile.component.ts mit, aber dort sind nicht die entsprechenden Anpassungen gemacht (siehe Zeile 30 in profile.component.ts) heißt es geschieht nichts damit.